src/unittest/test_mapdatabase.cpp

```diff
@@ -142,6 +143,7 @@ void TestMapDatabase::runTestsForCurrentDB()
 	TEST(testList, 1);
 	TEST(testRemove);
 	TEST(testList, 0);
+	TEST(testPositionEncoding);
```

Comment:
this runs it for every database type, which is unnecessary as `getIntegerAsBlock` are static functions.
you can move the `TEST()` to the top of `runTests` instead